src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order object) {
+        OrderResponseDto dto = new OrderResponseDto();
+        dto.setId(object.getId());
+        dto.setUser(object.getUser());
+        dto.setOrderDate(object.getOrderDate());
+        TicketResponseMapper mapper = new TicketResponseMapper();
```

Comment:
new TicketResponseMapper?? Dependency injection, Spring... don't we need them?))
And you can do it simplier 
![image](https://github.com/mate-academy/jv-spring-rest-web-practice/assets/20326858/54125e3e-06de-443d-9226-b5b20c01fae3)